Challange-1-iOS/Challange-1-iOS/Coordinator/Coordinator.swift

```diff
@@ -0,0 +1,8 @@
+import UIKit
+
+public protocol Coordinator: AnyObject {
```

Comment:
why is this `public` ?

Reply:
Don't need to be public, we can keep the variable as default (internal)